frontend/src/pages/PropertyDetails.jsx

```
@@ -11,6 +11,7 @@
 import PropTypes from 'prop-types';
 import { toast } from 'react-toastify';
 
+{/* Props and helper functions */}
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.

(lint/complexity/noUselessLoneBlockStatements)